language/German/strings.po

```diff
@@ -3474,7 +3535,7 @@ msgstr "Sound"
 
 #: src/frontend/mame/ui/sndmenu.cpp:149
 msgid "Compressor"
-msgstr ""
+msgstr "Komprimierer"
```

Comment:
in Audio, it's actually called "Kompressor" in German

Reply:
Fixed